tests/testTestMethods.py

```diff
@@ -0,0 +1,176 @@
+    """
+    def testAssertAnglesNearlyEqual(self):
+        """Test assertAnglesNearlyEqual"""
+        for angDeg in (0, 45, -75):
+            ang0 = angDeg*afwGeom.degrees
+            self.assertAnglesNearlyEqual(
+                ang0,
+                ang0 + 0.01*afwGeom.arcseconds,
+                maxDiff=0.010001*afwGeom.arcseconds,
+            )
+            self.assertAnglesNearlyEqual(
+                ang0,
+                ang0 - 0.01*afwGeom.arcseconds,
+                maxDiff=0.010001*afwGeom.arcseconds,
+            )
+            self.assertAnglesNearlyEqual(
+                ang0 - 720*afwGeom.degrees,
+                ang0 + 0.01*afwGeom.arcseconds,
+                maxDiff=0.010001*afwGeom.arcseconds,
+            )
+            self.assertAnglesNearlyEqual(
+                ang0,
+                ang0 + 360*afwGeom.degrees + 0.01*afwGeom.arcseconds,
+                maxDiff=0.010001*afwGeom.arcseconds,
+            )
+            self.assertRaises(AssertionError,
+                self.assertAnglesNearlyEqual,
+                    ang0,
+                    ang0 + 360*afwGeom.degrees + 0.010001*afwGeom.arcseconds,
+                    maxDiff=0.01*afwGeom.arcseconds,
+            )
+
+    def testAssertBoxesNearlyEqual(self):
+        """Test assertBoxesNearlyEqual"""
+        for min0 in ((0, 0), (-1000.5, 5000.1)):
+            min0 = afwGeom.Point2D(*min0)
+            for extent0 in ((2.01, 3.01), (5432, 2342)):
+                extent0 = afwGeom.Extent2D(*extent0)
+                box0 = afwGeom.Box2D(min0, extent0)
+                self.assertBoxesNearlyEqual(box0, box0, maxDiff=1e-7)
+                for deltaExtent in ((0.001, -0.001), (2, -3)):
+                    deltaExtent = afwGeom.Extent2D(*deltaExtent)
+                    box1 = afwGeom.Box2D(box0.getMin() + deltaExtent, box0.getMax())
+                    radDiff = math.hypot(*deltaExtent)
+                    self.assertBoxesNearlyEqual(box0, box1, maxDiff=radDiff*1.00001)
+                    self.assertRaises(AssertionError, self.assertBoxesNearlyEqual,
+                        box0, box1, maxDiff=radDiff*0.99999)
+
+                    box2 = afwGeom.Box2D(box0.getMin() - deltaExtent, box0.getMax())
+                    self.assertBoxesNearlyEqual(box0, box2, maxDiff=radDiff*1.00001)
+                    self.assertRaises(AssertionError, self.assertBoxesNearlyEqual,
+                        box0, box2, maxDiff=radDiff*0.999999)
+
+                    box3 = afwGeom.Box2D(box0.getMin(), box0.getMax() + deltaExtent)
+                    self.assertBoxesNearlyEqual(box0, box3, maxDiff=radDiff*1.00001)
+                    self.assertRaises(AssertionError, self.assertBoxesNearlyEqual,
+                        box0, box3, maxDiff=radDiff*0.999999)
+
+    def testAssertCoordsNearlyEqual(self):
+        """Test assertCoordsNearlyEqual"""
+        for raDecDeg in ((45, 45), (-70, 89), (130, -89.5)):
```

Comment:
I don't understand your question. assertAnglesNearlyEqual wraps the difference, by design and as documented. If you don't want to wrap the difference then you'd just use assertLess(abs(ang1-ang0), maxAng).